projects/cdk/types/mapper.ts

```diff
@@ -1,4 +1,11 @@
 /**
  * Typed mapping function.
+ *
+ * @deprecated Will be removed in v4.0. Use `TypedTuiMapper` instead.
  */
 export type TuiMapper<T, G> = (item: T, ...args: any[]) => G;
+
+/**
+ * Typed mapping function.
+ */
+export type TypedTuiMapper<T extends unknown[], G> = (...args: T) => G;
```

Comment:
Please change to `TuiTypedMapper` because our public imports should start with tui prefix.